docs/operations/misc/higherorder.jl

```diff
@@ -0,0 +1,28 @@
+# ---
+# title: Higher-order functions
+# description: a set of helper opeartions that allow applying any function
+# ---
```

Comment:
Looks like we need a cover image here.

Reply:
Is it required? I could not think of any since the operations are really general.

Reply:
No, but I guess it somehow doesn't look very nice if using the fallback cover... I really have a bad taste in designing the icon/cover. :(

Reply:
I will try to add a better cover picture in near future